src/main/java/com/adventofcode/day18/Day18.java

```diff
@@ -0,0 +1,67 @@
+package com.adventofcode.day18;
+
+import com.adventofcode.day06.Light;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class Day18 {
+
+    private Light[][] lights = new Light[100][100];
```

Comment:
You can get rid of the initialization (it gets overwritten either way) and then you can add final here